app/models/placement_group.rb

```diff
@@ -0,0 +1,81 @@
+class PlacementGroup < ApplicationRecord
+  acts_as_miq_taggable
+  include SupportsFeatureMixin
+  include NewWithTypeStiMixin
+  belongs_to :ext_management_system, :foreign_key => :ems_id, :class_name => "ManageIQ::Providers::CloudManager"
+  has_many :vms, :dependent => :nullify
+
+  delegate :queue_name_for_ems_operations, :to => :ext_management_system, :allow_nil => true
+
+  def self.class_by_ems(ext_management_system)
+    ext_management_system&.class_by_ems(:PlacementGroup)
+  end
+
+  def self.my_zone(ems)
+    # TODO(pblaho): find unified way how to do that
+    ems ? ems.my_zone : MiqServer.my_zone
+  end
+
+  def my_zone
+    self.class.my_zone(ext_management_system)
+  end
+
+  def self.create_placement_group_queue(userid, ext_management_system, options = {})
+    task_opts = {
+      :action => "creating Placement Group for user #{userid}",
+      :userid => userid
+    }
```

Comment:
I think you should raise an ArgumentError if `ext_management_system` is `nil` since you use it without checking below
```suggestion
  def self.create_placement_group_queue(userid, ext_management_system, options = {})
    raise ArgumentError, _("ext_management_system cannot be nil") if ext_management_system.nil?

    task_opts = {
      :action => "creating Placement Group for user #{userid}",
      :userid => userid
    }
```